docs/resources/node_room_member.md

```diff
@@ -24,6 +25,22 @@ resource "netdata_node_room_member" "test" {
     "node1",
     "node2"
   ]
+  rule {
+    action      = "INCLUDE"
+    description = "Description of the rule"
+    clause {
+      label    = "role"
+      operator = "equals"
+      value    = "parent"
+      negate   = false
+    }
```

Comment:
This is logic AND in such form?

Reply:
between rules is OR, between clauses is AND